pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -357,6 +360,11 @@ func (m *Manager) backgroundMetricsFlush(ctx context.Context) {
 	defer cleanUpTicker.Stop()
 	availableRUTicker := time.NewTicker(metricsAvailableRUInterval)
 	defer availableRUTicker.Stop()
+	recordMaxTicker := time.NewTicker(tickPerSecond)
+	defer recordMaxTicker.Stop()
+	maxPerSecTrackers := make(map[string]*maxPerSecCostTracker)
+	rruSum := make(map[string]float64)
+	wruSum := make(map[string]float64)
```

Comment:
what about move rruSum, wruSum into maxPerSecCostTracker, no need to maintain 3 maps.